Tests/TensorFlowTests/LayerTests.swift

```diff
@@ -361,6 +361,20 @@ final class LayerTests: XCTestCase {
         XCTAssertEqual(output, expected)
     }
 
+    func testDropout() {
+        Context.local.learningPhase = .training
+        let probability = 0.25
+        let layer = Dropout<Double>(probability: probability)
+        let input = Tensor<Double>(randomNormal: [500, 500], mean: Tensor(5))
+        let output = layer(input)
+        let fractionDropped = Tensor<Double>(output .== 0).mean()
+        XCTAssertTrue(fractionDropped.isAlmostEqual(to: Tensor(probability), tolerance: 0.01))
```

Comment:
```suggestion
        XCTAssertEqual(fractionDropped.scalar!, probability, tolerance: 0.01)
```
I forgot the exact argument label but `XCTAssertEqual` has an approximating variant.